Shortcuts/Utilities & Tools/files.json

```diff
@@ -0,0 +1,828 @@
+[
+  {
+    "groupInternalName": "Global",
+    "shortcuts": [
+      {
+        "internalName": "Toggle FullScreen mode",
+        "displayNames": {
+          "en-us": "Toggle FullScreen mode"
+        },
+        "keyCombination": [
+          [
+            "F11"
+          ]
+        ],
+        "searchKeywords": [
+          "Fullscreen"
+        ]
+      },
+      {
+        "internalName": "Enter compact overlay\r\n",
+        "displayNames": {
+          "en-us": "Enter compact overlay\r\n"
```

Comment:
Please avoid `\r\n`. Not sure how they got here. Maybe a bug in CtrlHelpEditor. 😅

Reply:
Indeed a bug in CtrlHelp Editor. Fixing it.

Reply:
Lol, I noticed this but thought it was by design. I pushed a fix.